gateway-ha/src/main/resources/gateway-ha-persistence-postgres.sql

```diff
@@ -75,4 +75,10 @@ CREATE TABLE IF NOT EXISTS exact_match_source_selectors (
 
     PRIMARY KEY (environment, source, query_type),
     UNIQUE (source, environment, query_type, resource_group_id)
-);
\ No newline at end of file
+);
+
+CREATE TABLE IF NOT EXISTS cookie_backend_lookup (
+    cookie VARCHAR(256) NOT NULL PRIMARY KEY,
```

Comment:
256 is long enough?

Reply:
It should be. The default session manager, which we are using, uses a long as session id: https://github.com/ganqing2017/jetty/blob/79b822339c3940679ac69a51eaed21b7af3e71c1/jetty-server/src/main/java/org/eclipse/jetty/server/session/DefaultSessionIdManager.java#L253